rkt/list.go

```diff
@@ -207,3 +210,22 @@ func fmtNets(nis []netinfo.NetInfo) string {
 	}
 	return strings.Join(parts, ", ")
 }
+
+func getImageName(p *pod, appName types.ACName) (string, error) {
+	aim, err := p.getAppsImageManifests()
```

Comment:
can we actually have something like `getAppImageManifest` to get a single image manifest?